src/Grphp/Client/Interceptors/Base.php

```diff
@@ -72,6 +72,46 @@ public function setRequest(&$request)
         $this->request = $request;
     }
 
+    /**
+     * Gets the fully qualified method name, e.g. grphp.catalog.products/GetProduct
+     *
+     * @return string
+     * @throws StubNotFoundException
+     */
+    public function getFullyQualifiedMethodName(): string
```

Comment:
do we want this to be the fully qualified name, or the fully qualified name that matches expected responses? 

`grphp.catalog.products/GetProduct` vs `grphp.catalog.products/getProduct`

In my local version had to lcfirst the method to match

Reply:
Yep, you're right, TAL

Reply:
So it looks like our generator actually generators them with the lowercase, e.g:

```
        'addAppCategory' => '\Bigcommerce\Rpc\Apps\AppCategoryResp',
        'removeAppCategory' => '\Bigcommerce\Rpc\Apps\AppCategoryResp',
```